src/adapters/beachfront.js

```diff
@@ -8,43 +8,60 @@ import { STATUS } from 'src/constants';
 const ENDPOINT = '//reachms.bfmio.com/bid.json?exchange_id=';
 
 function BeachfrontAdapter() {
-  var baseAdapter = Adapter.createNew('beachfront'),
-    bidRequest;
+  var baseAdapter = Adapter.createNew('beachfront');
 
-  baseAdapter.callBids = function (bidRequests) {
-    if (!bidRequests || !bidRequests.bids || bidRequests.bids.length === 0) {
-      return;
-    }
+  var totalBids = 0;
+  var attemptedBids = 0;
```

Comment:
This `attemptedBids`, `totalBids` logic is not going to work.  If you are involved in more than a single auction `totalBids` will be  set to `bids.length` while `attemptedBids` can keep getting larger resulting in you registering a bunch of `NO_BID`s.

Also, I'm not not sure what the purpose of counting the bids is to begin with.  Can you explain?